pages/national_resources/gr_resources.md

```diff
@@ -1,65 +1,48 @@
 ---
 
 title: Greece
-search_exclude: true
-description: <!---REPLACE THIS with a one sentence description of the page--->
+description: A comprehensive guide to research data management (RDM) in Greece, featuring tools, resources, and services tailored for the life sciences community.
 country_code: GR
-contributors: [<!---REPLACE THIS with comma separated list of contributors--->]
-coordinators: [<!---REPLACE THIS with the name of data management coordinators of your ELIXIR node--->]
+contributors: [Eleni Adamidi]
+coordinators: []
 
 # Link to other pages in the tool assembly section on the RDMkit by listing the page_id 
-related_pages:
-  tool_assembly: [<!---REPLACE THIS with the page ID of the tool_assembly pages that you want to list here as related pages--->]
-
-training:
-  - name: Training in TeSS
-    registry: TeSS
-    url: <!--- https://tess.elixir-europe.org/materials?node=NODENAME --->
-  - name: ELIXIR NODENAME community in Zenodo
-    registry: Zenodo
-    url: <!--- https://zenodo.org/communities/elixir NODENAME --->
-  - name: ELIXIR NODENAME YouTube
-    registry: YouTube
-    url: <!--- URL of the channel --->
-  - name: <!---REPLACE THIS with the name of your training in registry or platform--->
-    registry: <!---REPLACE THIS with the name of the registry--->
-    url: <!---REPLACE THIS with the url of your training registry or platform--->
 
 # Refer to entries of the "main_tool_ and_resource_table" if institutions, organizations and projects from the country contribute to the development of international tools and resources. 
-ref_to_main_resources: 
```

Comment:
@eleni-adamidi You can use this to list resources your country contributed to but is widely applicable. Add it to the main toolstable as described in out contribute section, and add it to a relevant page within the RDMkit (we can help with that). Then you can list here its tool ID.

Reply:
Lucky you it is already added! https://github.com/elixir-europe/rdmkit/blob/master/_data/tool_and_resource_list.yml#L175 just use the ID `argos`